### PR TITLE
add run description

### PR DESCRIPTION
**Pull request by jipiboily**
@epaulet

### CODE REVIEW

File: lib/integrations/slack.rb [hunk 1]
```diff
@@ -46,11 +46,11 @@ def message_color
     private
 
     def message_text
+      description = run[:description].nil? ? '' : "(#{run[:description]}) "
       {
-
-        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{run[:status]}.",
-        'run_error' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> errored: #{run[:error_reason]}.",
-        'run_webhook_timeout' => "Your Rainforest run <#{payload[:frontend_url]}|##{payload[:id]}> timed out due to your webhook failing. If you need a hand debugging it, please let us know via email at team@rainforestqa.com.",
+        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{description}#{run[:status]}.",
```
Comment (epaulet) [inline lib/integrations/slack.rb]: Descriptions all have spaces after them? There is no space/separation here between the description and the status.

Comment (jipiboily) [inline lib/integrations/slack.rb]: Nope, I've added it in https://github.com/rainforestapp/rainforest-integrations/pull/23/files#diff-1981ddaa8f271ef1cdef12f13ba34863R49

Maybe I should make that more obvious though...

Comment (jipiboily) [inline lib/integrations/slack.rb]: Is that making it clearer that way https://github.com/rainforestapp/rainforest-integrations/commit/f6375f07d9ca97d594743f63928eb7aa9fc796a6 ?

**Comment by epaulet** [comment]
@jipiboily Looks like run description was added in the other branch that was open. I missed the space earlier but I think it's clear, so I'll just close this branch for now.

**Pull request closed without merging**